app/src/app/components/transformation-view/transformation-view.component.ts

```diff
@@ -128,6 +128,14 @@ export class TransformationViewComponent implements OnInit, OnDestroy {
         result += 'unzip $folder.csar -d unzipped\n';
         result += 'bash unzipped/output/scripts/deploy.sh\n';
         result += 'cd ../\n';
+        result += 'printf "cleanup [y/N]: "\n' +
+            'read input\n' +
+            '\n' +
+            'if [ "$input" != "y" ] && [ "$input" != "yes" ] && [ "$input" != "Y" ]\n' +
```

Comment:
Don't be afraid of using regexp: `if [[ ! "$input" =~ ^(y|yes|Y)$ ]]; then`
Regex is awesome!